kontrol/postgres.go

```diff
@@ -415,6 +416,36 @@ func (p *Postgres) DeleteKey(keyPair *KeyPair) error {
 	return err
 }
 
+func (p *Postgres) IsValid(public string) error {
+	psql := sq.StatementBuilder.PlaceholderFormat(sq.Dollar)
```

Comment:
I've did explicitly so validity can be checked differently. For example we might check it via `deleted_at` field, someone else could made other checks. Just using `Get...` methods would be limited.